addOns/websocket/src/main/java/org/zaproxy/zap/extension/websocket/pscan/WebSocketPassiveScannerPlugin.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2018 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.websocket.pscan;
+
+import org.zaproxy.zap.utils.EnableableInterface;
+
+public class WebSocketPassiveScannerPlugin extends WebSocketPassiveScannerDecorator
+        implements EnableableInterface {
```

Comment:
We could remove this class and let `WebSocketPassiveScannerDecorator` implement `EnableableInterface` instead.

Reply:
This maybe lead us to rename `WebSocketPassiveScannerDecorator` to `WebSocketPassiveScannerPlugin`?

Reply:
I'd leave it as `WebSocketPassiveScannerDecorator`, it's still decorating a `WebSocketPassiveScanner`.

Reply:
Fixed in e37cf6b0a